src/app/api/services/dot-custom-event-handler/dot-custom-event-handler.service.ts

```diff
@@ -41,10 +41,10 @@ export class DotIframeEventsHandler {
      * Handle custom events from the iframe portlets
      *
      * @param CustomEvent event
-     * @memberof DotIframeEventsHandler
+     * @memberof DotCustomEventHandlerService
      */
     handle(event: CustomEvent): void {
-        if (this.handlers[event.detail.name]) {
+        if ( event && this.handlers[event.detail.name]) {
```

Comment:
Are we calling this without event?

Reply:
no, but in case the event we pass is undefined, just an extra check.